roles/ceph-mon/tasks/openstack_config.yml

```diff
@@ -62,3 +62,12 @@
   when:
     - openstack_config
     - cephx
+
+- name: chmod openstack key(s)
```

Comment:
isn't it possible to add `mode` attribute in the task above [L40](https://github.com/ceph/ceph-ansible/pull/1756/files#diff-9b4e69e1bd82ebfed75a8118c90977c1R40) ?

like:

```
 - name: copy to other mons the openstack key(s)
   copy:
     src: "{{ fetch_directory }}/{{ fsid }}/etc/ceph/{{ cluster }}.{{ item.1.name }}.keyring"
     dest: "/etc/ceph/{{ cluster }}.{{ item.1.name }}.keyring"
     mode: "{{ item.1.mode }}"
   with_nested:
     - "{{ groups[mon_group_name] }}"
     - "{{ openstack_keys }}"
   delegate_to: "{{ item.0 }}"
   when:
     - cephx
     - openstack_config
     - item.0 != groups[mon_group_name] | last
```